### PR TITLE
Fixed unordered lists in Scaled Registry and Application sections

### PR DESCRIPTION
**Pull request by ghost**
@openshift/team-documentation Please peer review.

### CODE REVIEW

File: scaling_performance/optimizing_storage.adoc [hunk 1]
```diff
@@ -120,8 +120,7 @@ In a scaled/HA {product-title} registry cluster deployment:
 
 * The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
 * File storage and block storage are not recommended for a scaled/HA {product-title} registry cluster deployment with production workloads.
-* All NAS storage (excluding CNS/CRS GlusterFS as it uses an object storage interface) are
-not recommended for {product-title} Registry cluster deployment with production workloads.
+* All NAS storage (excluding CNS/CRS GlusterFS as it uses an object storage interface) are not recommended for {product-title} Registry cluster deployment with production workloads.
```
Comment (ahardin-rh) [inline scaling_performance/optimizing_storage.adoc]: , as it uses

**Comment by ahardin-rh** [comment]
@tmorriso-rh just one nit from me. Otherwise, looks good! 🎆

**Comment by vikram-redhat** [comment]
@tmorriso-rh if for a BZ, please put the BZ number in the PR title.

**Comment by ahardin-rh** [comment]
@tmorriso-rh It looks like the issue is not in master and only in 3.6 and 3.7. This could have happened during a conflict resolution when cherry-picking to 3.6 and 3.7 branches. To fix the issue, what you will have to do is create a PR against enterprise-3.6-stage and enterpise-3.7-stage. I sent you an email with some more guidance just before shutdown (12/21). Let me know if you have any questions. Thanks!

**Comment by tmorriso-rh** [comment]
Work will continue in https://github.com/openshift/openshift-docs/pull/7030.

**Pull request closed without merging**